public/js/graph/react_graph.js

```diff
@@ -450,46 +536,59 @@ var Graph = React.createClass({
             preserveAspectRatio: 'xMinYMin'
         };
 
+        var zoomInDisabled = (this.state.zoomFactor <= 0.5) ? true : false;
```

Comment:
These expressions can all be simplified - `zoomInDisabled = this.state.zoomFactor <= 0.5`